models_evaluation.py

```diff
@@ -0,0 +1,176 @@
+from os import listdir
+from os.path import isfile, join
+import sys
+import argparse
+import logging
+import pathlib
+
+import gensim
+import pandas as pd
+import itertools
+
+logger = logging.getLogger("fasttext")
+
+
+def questions_reading(analogies):
+    """
+    Reads all type of questions to dict.
+    :param analogies: string
+    :return: dict
+    """
+    analogy_dict = dict()
+    with open(analogies, "r") as analogy_f:
+        key = None
+        for line in analogy_f:
+            if line in ['\n', '\r\n']:
+                continue
+            if line.startswith(":"):
+                # new section starts
+                key = line.rstrip()
+                analogy_dict[key] = list()
+            else:
+                analogy_dict[key].append(line[:-1].split("\t"))
```

Comment:
:-1 is to remove a newline character?
if so, it's better to do something like:

`for line in map(str.strip, analogy_f):` at the top of the loop.